actions/nix-build/action.yml

```diff
@@ -0,0 +1,142 @@
+name: nix-build
+description: "Builds all manifest models and uploads them to a specified cache"
+
+inputs:
+  # general inputs --------------------------------
+  add-packages:
+    description: "Additional packages to build on top of the declared check packages (space separated) - not used if setup-only=true"
+    required: false
+    default: ""
+
+  # cache inputs ----------------------------------
+  cache-url:
+    description: "Nix cache URL"
+    required: false
+    default: ""
+  cache-privkey:
+    description: "Nix cache binary private signing key - not used if setup-only=true"
+    required: false
+    default: ""
+
+  # aws inputs --------------------------
+  # see https://github.com/aws-actions/configure-aws-credentials/blob/main/action.yml
+  aws-region:
+    description: "AWS region"
+    required: false
+    default: us-west-2
```

Comment:
could we make this required and not expose the default value? we usually pass this a secret.